sopel/modules/url.py

```diff
@@ -64,6 +76,14 @@ def configure(config):
         ' shorter version of the URL in the title as a TinyURL link'
         ' (0 to disable)'
     )
+    config.url.configure_setting(
+        'enable_private_resolution',
+        'Enables URL lookups for RFC1918 addresses?'
```

Comment:
`'Enable**s** URL lookups for RFC1918 addresses?'` :scream: but dat ess tho!

Reply:
Lol. Fixed.